content/en/docs/reference/glossary/cloud-provider.md

```diff
@@ -4,15 +4,29 @@ id: cloud-provider
 date: 2018-04-12
 full_link: /docs/concepts/cluster-administration/cloud-providers
 short_description: >
-  Cloud provider is a company that offers cloud computing platform that can run Kubernetes clusters.
+  An organization that offers a cloud computing platform.
 
-aka: 
+aka:
+- Cloud Service Provider
 tags:
 - community
 ---
- Cloud provider is a company that offers cloud computing platform that can run Kubernetes clusters.
+ A business or other organization that offers a cloud computing platform.
 
-<!--more--> 
+<!--more-->
 
-Cloud providers or sometime called Cloud Service Provider (CSPs) provides cloud computing platforms.  They may offer services such as Infrastructure as a Service (IaaS) or Platform as a Service (PaaS).  Cloud providers host the Kubernetes cluster and also provide services that interact with the cluster, such as Load Balancers, Storage Classes etc. 
+Cloud providers, sometimes called Cloud Service Providers (CSPs), offer
+cloud computing platforms or services.
 
+Many cloud providers offer managed infrastructure (also called
+Infrastructure as a Service or IaaS).
+With managed infrastructure the cloud provider is responsible for
+servers, storage, and networking while you manage layers on top of that
+such as running a Kubernetes cluster.
+
+You can also find Kubernetes as a managed service; sometimes called
```

Comment:
@sftim , **this is a nit**: Looking for symmetry with line 23, as you are now writing about managed platforms:
Suggested replace `You can also find Kubernetes` with something like:
Some cloud providers also offer Kubernetes as a managed service (sometimes referred to as  Platform as a Service or PaaS).